src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -276,7 +276,7 @@ JNI_METHOD(void, beginConnectDeviceIp)(JNIEnv * env, jobject self, jlong deviceC
     ChipLogProgress(Controller, "beginConnectDevice() called with IP Address");
 
     const char * deviceAddrStr = env->GetStringUTFChars(deviceAddr, 0);
-    chip::Inet::IPAddress::FromString(deviceAddrStr, deviceIPAddr);
+    deviceIPAddr.FromString(deviceAddrStr, deviceIPAddr);
```

Comment:
It seems to me that it is a Doxygen parsing issue. Basically doxygen output an error saying that it can find the class definition but it proposes me the exact right candidates for it. I believe the parser is in a weird state for this file since the following error you mentioned is also something that confuses doxygen.